src/careamics/models/lvae/noise_models.py

```diff
@@ -34,20 +34,76 @@ def noise_model_factory(
         If the chosen noise model `model_type` is not implemented.
         Currently only `GaussianMixtureNoiseModel` is implemented.
     """
-    if model_config:
+    if model_config.noise_models:
```

Comment:
Wouldn't this raise an error if `model_config` is `None` (e.g., `NoneType` does not have attribute `noise_models`)? Remember that the config is None, for instance, in the case of muSplit, where we instantiate the lghtning module without requiring the noise model.

Reply:
P.S. I'd rather have this instead of instantiating a `MultiChannelNMConfig` with empty `noise_models` list like you do in the test. Indeed, it is more coherent with the fact that we do not actually have a noise model in some cases (e.g., muSplit).

Reply:
I can make an example to support my statement:
Suppose we initialize a lightning module for `muSplit`. If one wants to have a look to the internal attributes of the model, it would be way more clean and coherent to have the noise model set to `None`, since in this case it is not needed, and, hence, not defined. Having an empty noise model would make things less linear in my opinion. 

Plus I'd need to refactor the `training_step` code that is coming later, which would be painful since it is a lot of code.